Communitter/src/main/java/com/communitter/api/controller/PostController.java

```diff
@@ -29,4 +33,14 @@ public ResponseEntity<String> deletePost(@PathVariable Long id){
         return ResponseEntity.ok(postService.deletePost(id));
     }
 
+    @PreAuthorize("@authorizer.checkSubscription(#root,#id)")
+    @PostMapping("/vote/{id}")
```

Comment:
{id} vote id gibi algılanıyor. posts/{id}/vote daha uygun olabilir.

Reply:
Thanks for the hint. It is fixed.